tests/PHPStan/Analyser/data/bug-3019.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace Bug3019;
+
+use function PHPStan\Testing\assertType;
+
+trait FooTrait
+{
+	public function doFoo(): void
+	{
+		assertType('string', __CLASS__);
```

Comment:
I am wondering that we don't get a class-string here

Reply:
My guess it's not checked because a duplicate trait with the same name exists.